cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,51 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
 
-  });
+import SettingsPageObject from '../support/pages/settings.pageObject';
 
-  beforeEach(() => {
+const settings = new SettingsPageObject();
+
+describe('Settings page', () => {
+  let user;
 
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('testData').then(testData => {
+      user = testData;
+    });
+    cy.login();
+    settings.visit();
   });
 
   it('should provide an ability to update username', () => {
-
+    settings.usernameSettings.clear().type(user.newUsername);
+    settings.updateBtnSettings.click();
```

Comment:
you will always click the button in tests. to optimize code you can move the click into the method and rename the method to say what it does clickSettingsBtn() whatnot